components/eamxx/src/physics/shoc/impl/shoc_assumed_pdf_compute_buoyancy_flux_impl.hpp

```diff
@@ -0,0 +1,41 @@
+#ifndef SHOC_SHOC_ASSUMED_PDF_COMPUTE_BUOYANCY_FLUX_IMPL_HPP
+#define SHOC_SHOC_ASSUMED_PDF_COMPUTE_BUOYANCY_FLUX_IMPL_HPP
+
+#include "shoc_functions.hpp" // for ETI only but harmless for GPU
+
+#include <iomanip>
+
+namespace scream {
+namespace shoc {
+
+/*
+ * Implementation of shoc_assumed_pdf_compute_buoyancy_flux. Clients should NOT
+ * #include this file, but include shoc_functions.hpp instead.
+ */
+
+template<typename S, typename D>
+KOKKOS_FUNCTION
```

Comment:
Given how tiny these functions are, we should probably use KOKKOS_INLINE_FUNCTION. Although the compiler will likely inline them anyways, we don't want to risk them to not be inlined.